public/app/plugins/panel/table/table-new/cells/TextWrapOptionsEditor.tsx

```diff
@@ -0,0 +1,29 @@
+import { selectors } from '@grafana/e2e-selectors';
+import { t } from '@grafana/i18n';
+import { TableCellOptions, TableWrapTextOptions } from '@grafana/schema';
+import { Field, Switch } from '@grafana/ui';
+
+import { TableCellEditorProps } from '../TableCellOptionEditor';
+
+export const TextWrapOptionsEditor = ({
+  cellOptions,
+  onChange,
+}: TableCellEditorProps<TableCellOptions & TableWrapTextOptions>) => {
+  // Handle row coloring changes
+  const onWrapTextChange = () => {
+    cellOptions.wrapText = !cellOptions.wrapText;
+    onChange(cellOptions);
+  };
+
+  return (
```

Comment:
The changes I've made to the types for textWrap as well as this change to `AutoCellOptionsEditor` becoming `TextWrapOptionsEditor` are setting up the max wrap length coming to everything that has text wrapping.